addons/website_slides_survey/models/slide_slide.py

```diff
@@ -64,14 +64,13 @@ def _compute_slide_type(self):
             if slide.slide_category == 'certification':
                 slide.slide_type = 'certification'
 
-    @api.model
-    def create(self, values):
-        rec = super(Slide, self).create(values)
-        if rec.survey_id:
-            rec.slide_category = 'certification'
-        if 'survey_id' in values:
-            rec._ensure_challenge_category()
-        return rec
+    @api.model_create_multi
+    def create(self, vals_list):
+        slides = super().create(vals_list)
+        slides_with_survey = slides.filtered(lambda s: s.survey_id)
```

Comment:
`filtered`, just like `mapped`, supports field names as arguments
```suggestion
        slides_with_survey = slides.filtered('survey_id')
```